modules/services/sql/variables.tf

```diff
@@ -6,6 +6,9 @@ variable "engine_version" {
   default = "5.6.48"
 }
 variable "allocated_storage" {}
+variable "max_allocated_storage" {
+  default = 100
+}
```

Comment:
Could you add a comment that indicates that this [enables autoscaling when `allocated_storage` < `max_allocated_storage`](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/db_instance#storage-autoscaling)?

That way, anyone who's using this template will know how to disable autoscaling if they want to.

Reply:
On second thought, these docs should go in the [README](https://github.com/open-craft/terraform-scripts/tree/main/modules/services/sql#input)

Reply:
I updated the readme, I think it's actually that it needs to be 0 to be disabled :+1:

Reply:
Ah cool yes, the terraform docs agree; 0 disables autoscaling. Thanks for updating the docs!